source/classes.tex

```diff
@@ -2118,10 +2136,25 @@
 same class as the \grammarterm{nested-name-specifier}.
 \end{itemize}
 
-A destructor shall take no arguments\iref{dcl.fct}.
+A prospective destructor shall take no arguments\iref{dcl.fct}.
```

Comment:
What is a "prospective destructor"???  We never define it - can we please?

Reply:
Much like how the prior state of this subclause ([class.dtor]) was to define the term "destructor" by describing the properties of such a thing, this subclause now defines "prospective destructor" by describing its properties. Effectively this subclause  *is* the definition of "prospective destructor".

Reply:
That said, I certainly wouldn't mind wrapping the first mention of "prospective destructor" in paragraph 1 with `\defn`.

Reply:
I would like to see a proper `\defnadj{prospective}{destructor}` definition.  I know it wasn't in the paper, but it should have been.  Should be easy enough to fix it editorially.

Reply:
Done, and I added the missing definition for "constructor" too (which used the same phrasing as we have here).